src/midi/midi_mt32.cpp

```diff
@@ -62,37 +63,126 @@ constexpr bool USE_NICE_RAMP = true;
+                                        &mt32_bluer_model, &mt32_1_06_model,
+                                        &mt32_1_05_model,  &mt32_1_04_model};
+
 MidiHandler_mt32 mt32_instance;
 
 static void init_mt32_dosbox_settings(Section_prop &sec_prop)
 {
 	constexpr auto when_idle = Property::Changeable::WhenIdle;
 
-	const char *models[] = {"auto", "cm32l", "mt32", 0};
+	const char *models[] = {"auto",
+	                        cm32l_any_model.Name().c_str(),
+	                        cm32l_1_02_model.Name().c_str(),
+	                        cm32l_1_00_model.Name().c_str(),
+	                        mt32_any_model.Name().c_str(),
+	                        mt32_new_model.Name().c_str(),
+	                        mt32_2_04_model.Name().c_str(),
+	                        mt32_old_model.Name().c_str(),
+	                        mt32_1_07_model.Name().c_str(),
+	                        mt32_bluer_model.Name().c_str(),
+	                        mt32_1_06_model.Name().c_str(),
+	                        mt32_1_05_model.Name().c_str(),
+	                        mt32_1_04_model.Name().c_str(),
+	                        0};
 	auto *str_prop = sec_prop.Add_string("model", when_idle, "auto");
 	str_prop->Set_values(models);
 	str_prop->Set_help(
-	        "Model of synthesizer to use. The default (auto) prefers CM-32L\n"
-	        "if both sets of ROMs are provided. For early Sierra games and Dune 2\n"
-	        "it's recommended to use 'mt32', while newer games typically made\n"
-	        "use of the CM-32L's extra sound effects (use 'auto' or 'cm32l')");
+	        "Model of synthesizer to use.\n"
+	        "'auto' picks the first model with available ROMs, in order as listed.\n"
+	        "'mt32_old' and 'mt32_new' are aliases for 1.07 and 2.04, respectively.");
 
 	str_prop = sec_prop.Add_string("romdir", when_idle, "");
 	str_prop->Set_help(
-	        "The directory containing one or both pairs of MT-32 and/or CM-32L ROMs.\n"
-	        "The files must be named in capitals, as follows:\n"
-	        "  - MT-32 ROM pair: MT32_CONTROL.ROM and MT32_PCM.ROM\n"
-	        "  - CM-32L ROM pair: CM32L_CONTROL.ROM and CM32L_PCM.ROM\n"
+	        "The directory containing ROMs for one or more models.\n"
 	        "The directory can be absolute or relative, or leave it blank to\n"
 	        "use the 'mt32-roms' directory in your DOSBox configuration\n"
-	        "directory, followed by checking other common system locations.");
+	        "directory, followed by checking other common system locations.\n"
+	        "ROM files inside this directory must be named as follows:\n"
+	        "  - MT32_CONTROL.ROM and MT32_PCM.ROM, for model 'mt32'\n"
+	        "  - CM32L_CONTROL.ROM and CM32L_PCM.ROM for model 'cm32l'\n"
+	        "  - Unzipped MAME MT-32 and CM-32L ROMs, to use the versioned models.\n");
 }
 
 #if defined(WIN32)
 
 static std::deque<std::string> get_rom_dirs()
 {
 	return {
+	        "..\\mt32-roms\\",
```

Comment:
I don't like going up the directory when looking for the ROMs… for soundfonts we let users do it themselves if they want to…

If such change is really needed, it indicates we probably should make "mt32-roms" directory configurable like "captures" directory?

Reply:
I guess it's just a personal preference. 
I've always tried to make my game collections almost stand-alone, and organized like:

```
   Game A/
   Game B/
   Game C/
   ...
   Game Z/
   Supporting Soundfonts/
   Supporting ROMs/
```

Because I didn't like have supporting files spread elsewhere.

But making it user-configurable is OK as well. (To be consistent, I would also prefer ../soundfonts/ also be searched :-)

Reply:
Will knock this out. 

Makes more sense to consult w/ the community to see if there's a common layout folks are using - and if there's anything better we can do to accommodate.